frontend/app/templates/tasks.hbs

```diff
@@ -1,5 +1,9 @@
+{{navigation-bar session=session}}
```

Comment:
We might want to link to the `login` or `password recovery` screens in a user case. Furthermore, we do want to display the navbar when showing a task, even if the user is not logged in

In any case, I think the logic is too complicated to get into these details now, that we have only one template requiring the navbar (we don't even have the profile screen designed. What would edit the user in the user details? The email? Does that mean that we are changing them from all the turns she is responsible of?)

If the app grows, and we had more use cases and experience, I would get into these details, but I would keep it simple for now